tests/frontend/specs/timeslider_follow.js

```diff
@@ -0,0 +1,52 @@
+describe("timeslider", function(){
+  //create a new pad before each test run
+  beforeEach(function(cb){
+    helper.newPad(cb);
+    this.timeout(6000);
+  });
+
+  it("follow content as it's added to timeslider", function(done) { // passes
+    var inner$ = helper.padInner$;
+    var chrome$ = helper.padChrome$;
+
+    // make some changes to produce 100 revisions
+    var timePerRev = 900
+      , revs = 5;
+    this.timeout(revs*timePerRev+10000);
+    for(var i=0; i < revs; i++) {
+      setTimeout(function() {
+        // enter 'a' in the first text element
+        inner$("div").last().sendkeys('a\n');
+        inner$("div").last().sendkeys('{enter}');
+      }, timePerRev*i);
+    }
+
+    setTimeout(function() {
+      // go to timeslider
+      $('#iframe-container iframe').attr('src', $('#iframe-container iframe').attr('src')+'/timeslider#0');
+
+      setTimeout(function() {
+        var timeslider$ = $('#iframe-container iframe')[0].contentWindow.$;
+        var $sliderBar = timeslider$('#ui-slider-bar');
+
+        var latestContents = timeslider$('#innerdocbody').text();
+
+        // set to follow contents as it arrives
+        timeslider$('#options-followContents').prop("checked", true);
+
+        var originalTop = timeslider$('#innerdocbody').offset();
+        timeslider$('#playpause_button_icon').click();
+
+        setTimeout(function() {
+          //make sure the text has changed
+          var newTop = timeslider$('#innerdocbody').offset();
+          expect( originalTop ).not.to.eql( newTop );
+          done();
+        }, 1000);
+
+      }, 2000);
+    }, revs*timePerRev);
```

Comment:
Maybe I misunderstand, but does this work and test what you wanted to test? I think this waits until all edits have been made before entering timeslider, so it won't really follow new edits?

We probably need special test equipment for this to work, eg two browsers (one in timeslider, one doing the edits) like in https://webdriver.io/docs/multiremote.html

I think we can hack this together with wd, too: we need access to known pad ids, but we can set them via existing code (like we do with specs_list) during test setup. Then, at the beginning or end of the tests we wait in those "special" pads and run our tests as soon as other browser join. Of course this needs more thinking - should the browsers roll the dice to find out who edits the pad and who watches the edits in the timeslider? Should they switch roles so that we can test if the code works across browsers? We probably also want to handle local testing with multiple browsers..